cextern/wcslib/CHANGES

```diff
@@ -1,3 +1,17 @@
+WCSLIB version 5.10 (2015/10/09)
+--------------------------------
+
+* C library
+
+  - In wcshdo(), allow output of floating point keyvalues to 15
+    significant digits (was 14), mainly for astropy.
+
+  - In wcspih(), allow unrecognised WAT projection types (tan, etc.) to
+    pass without returning an error.  Reported by Michael Droettboom.
+    Also fixed a bug triggered by SIP keywords in unconventional order,
+    reported by Colin Slater, LSST.
```

Comment:
Yes, I've confirmed this fixes #4089. Any chance it can go in 1.1? Or is it too late?
 @embray @astrofrog